Forge.TreeWalker.UnitTests/test/ExecutorUnitTests.cs

```diff
@@ -185,15 +186,15 @@ public void TestExecutor_Success_CompileExpressionWithForgeDefaultDependenciesBe
             dependencies.Add(typeof(Type));
 
             // Default dependencies are expected to be tossed away internally in ExpressionExecutor. 
-            ExpressionExecutor ex = new ExpressionExecutor(null, this.UserContext, null);
+            ExpressionExecutor ex = new ExpressionExecutor(null, this.UserContext, null, null);
```

Comment:
I've updated the overloads. I was initially avoiding adding scriptCache to the parameters because even if you pass in null, you need to reference all the Roslyn packages. But it made the overloads look weird.

I've bit the bullet on that and have included the Roslyn packages in UnitTests csproj. I've amended the overloads to look better.